data/foss2cyber.json

```
@@ -0,0 +1,63 @@
+{
+    "name": "Ishan Velle",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Front-end Web Developer | Self-taught | Designer | Freelancer | Blogger",
+    "avatar": "https://github.com/foss2cyber.png"
+    "tags": [
+      "FOSS",
+      "Web Developement",
+      "Freelancing",
+      "GNU/Linux",
+      "Homelab",
+      "Self-hosting",
+      "Infosec",
+      "OSINT/DFIR",
```

Please remove this extra comma `,` from here to validate your json. 

```suggestion
      "OSINT/DFIR"
```